CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+# Contributing to `linter`
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to linter.
+These are just guidelines, not rules, use your best judgment and feel free to
+propose changes to this document in a pull request.
+
+This project adheres to the [Open Code of Conduct][code-of-conduct]. By participating, you are expected to uphold this code.
+[code-of-conduct]: http://todogroup.org/opencodeofconduct/#Atom/opensource@github.com
+
+## Submitting Issues
+
+* You can create an issue [here](https://github.com/atom-community/linter/issues/new), but
+  before doing that please read the notes below on collecting information and submitting issues.
+  Include as many details as possible with your report.
+* Include the version of Atom you are using and the OS.
+* Include screenshots and animated GIFs whenever possible; they are immensely
+  helpful.
+* Include the behavior you expected and other places you've seen that behavior
+  such as Emacs, vi, Xcode, etc.
+* Check the dev tools (`alt-cmd-i`) for errors to include. If the dev tools
+  are open _before_ the error is triggered, a full stack trace for the error
+  will be logged. If you can reproduce the error, use this approach to get the
+  full stack trace and include it in the issue.
+* On Mac, check Console.app for stack traces to include if reporting a crash.
+* Perform a [cursory search](https://github.com/atom-community/linter/search?q=&type=Issues&utf8=%E2%9C%93)
+  to see if a similar issue has already been submitted.
+* Please setup a [profile picture](https://help.github.com/articles/how-do-i-set-up-my-profile-picture)
+  to make yourself recognizable and so we can all get to know each other better.
```

Comment:
I actually really love this. It's a lot easier talking to someone who I can associate with a face/emblem.